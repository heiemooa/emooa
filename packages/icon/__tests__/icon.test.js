const OLD_NODE_ENV = process.env.NODE_ENV;
process.env.NODE_ENV = 'development';
const icon = require('@emooa/icon');

describe('icon', () => {
  afterAll(() => {
    process.env.NODE_ENV = OLD_NODE_ENV;
  });

  it('exports modules correctly', () => {
    expect(Object.keys(icon)).toMatchSnapshot();
  });
});
