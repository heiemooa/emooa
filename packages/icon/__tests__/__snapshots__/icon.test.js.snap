// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon exports modules correctly 1`] = `
[
  "IconAlignCenter",
  "IconAlignLeft",
  "IconAlignRight",
  "IconAlipay",
  "IconAlipayCircleFill",
  "IconAliwangwangFill",
  "IconAndroid",
  "IconAndroidFill",
  "IconAppAdd",
  "IconApple",
  "IconAppleFill",
  "IconApps",
  "IconArchive",
  "IconArrowDown",
  "IconArrowDownFill",
  "IconArrowLeft",
  "IconArrowRight",
  "IconArrowUp",
  "IconArrowUpFill",
  "IconAt",
  "IconAttachment",
  "IconBackward",
  "IconBehance",
  "IconBehanceCircleFill",
  "IconBgColors",
  "IconBing",
  "IconBold",
  "IconBook",
  "IconBranch",
  "IconBrush",
  "IconBug",
  "IconBulb",
  "IconBytedanceColor",
  "IconCalendar",
  "IconCalendarClock",
  "IconCamera",
  "IconCaretDownFill",
  "IconCaretLeftFill",
  "IconCaretRightFill",
  "IconCaretUpFill",
  "IconCheck",
  "IconCheckCircle",
  "IconCheckCircleFill",
  "IconCheckSquare",
  "IconChineseFill",
  "IconChrome",
  "IconChromeColor",
  "IconChromeFill",
  "IconClockCircle",
  "IconClose",
  "IconCloseCircle",
  "IconCloseCircleFill",
  "IconCloud",
  "IconCloudDownload",
  "IconCode",
  "IconCodeBlock",
  "IconCodeSandbox",
  "IconCodeSquare",
  "IconCodepen",
  "IconCodepenCircleFill",
  "IconCommand",
  "IconCommon",
  "IconCompass",
  "IconCopy",
  "IconCopyright",
  "IconCustomerService",
  "IconDashboard",
  "IconDelete",
  "IconDesktop",
  "IconDice",
  "IconDingdingFill",
  "IconDoubleDown",
  "IconDoubleLeft",
  "IconDoubleRight",
  "IconDoubleUp",
  "IconDown",
  "IconDownCircle",
  "IconDownload",
  "IconDragArrow",
  "IconDragDot",
  "IconDragDotVertical",
  "IconDriveFile",
  "IconEar",
  "IconEdit",
  "IconEmail",
  "IconEmpty",
  "IconEnglishFill",
  "IconEraser",
  "IconExclamation",
  "IconExclamationCircle",
  "IconExclamationCircleFill",
  "IconExclamationPolygonFill",
  "IconExpand",
  "IconExperiment",
  "IconExport",
  "IconEye",
  "IconEyeInvisible",
  "IconFaceBookCircleFill",
  "IconFaceFrownFill",
  "IconFaceMehFill",
  "IconFaceSmileFill",
  "IconFacebook",
  "IconFacebookSquare",
  "IconFacebookSquareFill",
  "IconFile",
  "IconFileAudio",
  "IconFileImage",
  "IconFilePdf",
  "IconFileVideo",
  "IconFilter",
  "IconFindReplace",
  "IconFire",
  "IconFolder",
  "IconFolderAdd",
  "IconFolderDelete",
  "IconFontColors",
  "IconFormula",
  "IconForward",
  "IconFullscreen",
  "IconFullscreenExit",
  "IconGift",
  "IconGithub",
  "IconGithubFill",
  "IconGitlab",
  "IconGoogle",
  "IconGoogleCircleFill",
  "IconGoogleMapSquareFill",
  "IconGooglePlusCircleFill",
  "IconH1",
  "IconH2",
  "IconH3",
  "IconH4",
  "IconH5",
  "IconH6",
  "IconH7",
  "IconHeart",
  "IconHeartFill",
  "IconHighlight",
  "IconHistory",
  "IconHome",
  "IconHtml5",
  "IconIdcard",
  "IconIe",
  "IconImage",
  "IconImageClose",
  "IconImport",
  "IconInfo",
  "IconInfoCircle",
  "IconInfoCircleFill",
  "IconInstagram",
  "IconInstagramFill",
  "IconInteraction",
  "IconItalic",
  "IconLanguage",
  "IconLarkColor",
  "IconLaunch",
  "IconLayout",
  "IconLeft",
  "IconLeftCircle",
  "IconLineHeight",
  "IconLink",
  "IconLinkedin",
  "IconList",
  "IconLiveBroadcast",
  "IconLoading",
  "IconLocation",
  "IconLock",
  "IconLoop",
  "IconMan",
  "IconMenu",
  "IconMenuFold",
  "IconMenuUnfold",
  "IconMessage",
  "IconMessageBanned",
  "IconMindMapping",
  "IconMinus",
  "IconMinusCircle",
  "IconMinusCircleFill",
  "IconMobile",
  "IconMoon",
  "IconMoonFill",
  "IconMore",
  "IconMoreVertical",
  "IconMosaic",
  "IconMusic",
  "IconMute",
  "IconMuteFill",
  "IconNav",
  "IconNotification",
  "IconNotificationClose",
  "IconObliqueLine",
  "IconOrderedList",
  "IconOriginalSize",
  "IconPalette",
  "IconPaste",
  "IconPause",
  "IconPauseCircle",
  "IconPauseCircleFill",
  "IconPen",
  "IconPenFill",
  "IconPhone",
  "IconPlayArrow",
  "IconPlayArrowFill",
  "IconPlayCircle",
  "IconPlayCircleFill",
  "IconPlus",
  "IconPlusCircle",
  "IconPlusCircleFill",
  "IconPoweroff",
  "IconPrinter",
  "IconPublic",
  "IconPushpin",
  "IconQq",
  "IconQqCircleFill",
  "IconQqFill",
  "IconQqZone",
  "IconQrcode",
  "IconQuestion",
  "IconQuestionCircle",
  "IconQuestionCircleFill",
  "IconQuote",
  "IconRecord",
  "IconRecordStop",
  "IconRedo",
  "IconRefresh",
  "IconReply",
  "IconRight",
  "IconRightCircle",
  "IconRobot",
  "IconRobotAdd",
  "IconRotateLeft",
  "IconRotateRight",
  "IconSafe",
  "IconSave",
  "IconScan",
  "IconSchedule",
  "IconScissor",
  "IconSearch",
  "IconSelectAll",
  "IconSend",
  "IconServerless",
  "IconSettings",
  "IconShake",
  "IconShareAlt",
  "IconShareExternal",
  "IconShareInternal",
  "IconShrink",
  "IconSketch",
  "IconSkin",
  "IconSkipNext",
  "IconSkipNextFill",
  "IconSkipPrevious",
  "IconSkipPreviousFill",
  "IconSkypeFill",
  "IconSort",
  "IconSortAscending",
  "IconSortDescending",
  "IconSound",
  "IconSoundFill",
  "IconStamp",
  "IconStar",
  "IconStarFill",
  "IconSteamCircleFill",
  "IconStop",
  "IconStorage",
  "IconStrikethrough",
  "IconSubscribe",
  "IconSubscribeAdd",
  "IconSubscribed",
  "IconSun",
  "IconSunFill",
  "IconSwap",
  "IconSync",
  "IconTag",
  "IconTags",
  "IconTaobao",
  "IconTaobaoCircleFill",
  "IconTelegram",
  "IconTelegramCircleFill",
  "IconThumbDown",
  "IconThumbDownFill",
  "IconThumbUp",
  "IconThumbUpFill",
  "IconThunderbolt",
  "IconTiktok",
  "IconTiktokColor",
  "IconTiktokSquare",
  "IconToBottom",
  "IconToLeft",
  "IconToRight",
  "IconToTop",
  "IconTool",
  "IconTranslate",
  "IconTrophy",
  "IconTwitter",
  "IconTwitterCircleFill",
  "IconTwitterFill",
  "IconTwitterSquareFill",
  "IconUnderline",
  "IconUndo",
  "IconUnlock",
  "IconUnorderedList",
  "IconUp",
  "IconUpCircle",
  "IconUpload",
  "IconUser",
  "IconUserAdd",
  "IconUserGroup",
  "IconVercel",
  "IconVercelCircleFill",
  "IconVercelFill",
  "IconVideoCamera",
  "IconVoice",
  "IconWechat",
  "IconWechatFill",
  "IconWechatpay",
  "IconWechatpayFill",
  "IconWeibo",
  "IconWeiboCircleFill",
  "IconWifi",
  "IconWindows",
  "IconWindowsFill",
  "IconWoman",
  "IconXiguaColor",
  "IconXunleiFill",
  "IconYahoo",
  "IconYoutube",
  "IconYoutubeFill",
  "IconYuque",
  "IconYuqueFill",
  "IconZhihu",
  "IconZhihuCircleFill",
  "IconZoomIn",
  "IconZoomOut",
]
`;
